pages/linux/conky.md

```diff
@@ -0,0 +1,27 @@
+# conky
+
+> Light-weight system monitor for X.
+
+- Launch with default, built-in config:
+
+`conky`
+
+- Create a new, default config:
+
+`conky -C > ~/.conkyrc`
+
+- Load a config file:
+
+`conky -c {{/path/to/config}}`
+
+- Start in the background (daemonize):
+
+`conky -d`
+
+- Align specified config file on screen:
+
+`conky -c {{path/to/config}} -a {{{top,bottom,middle}_{left,right,middle}}}`
```

Comment:
Is the final format like `{1,2,3_4,5,6}`? If Not, I think it's fine to only have tokens around the 2 parts maybe. Rather than around the whole thing.

Reply:
Yes, the final format is `{top,bottom,middle}_{left,right,middle}` with an underline (a token).

Reply:
Oh, then this is fine. 👍